plugins/circleci-heroku/readme.md

```diff
@@ -0,0 +1,43 @@
+# `circleci-heroku` Tool Kit plugin
+
+WIP
+
+## Installation
+
+Install `@dotcom-tool-kit/circleci-heroku` as a `devDependency` in your app:
+
+```sh
+npm install --save-dev @dotcom-tool-kit/circleci-heroku
+```
+
+Add the plugin to your [Tool Kit configuration](https://github.com/financial-times/dotcom-tool-kit/blob/main/readme.md#configuration):
+
+```yaml
+plugins:
+	- '@dotcom-tool-kit/circleci-heroku'
+```
+
+### For Tool Kit generated Circle `config.yml`
+
+If you are migrating your project to Tool Kit for the first time then this plugin can generate a new `.circleci/config.yml` file for your project including Tool Kit configured workflows. To use this feature please delete or rename your existing Circle `config.yml` file before running the install command.
```

Comment:
**suggestion:** We should have this lower down in the readme under a heading like "migrating for first time".